src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -501,6 +510,7 @@ private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
                 "${courseId}", SanitizationHelper.sanitizeForHtml(course.getId()),
                 "${feedbackSessionName}", SanitizationHelper.sanitizeForHtml(session.getFeedbackSessionName()),
                 "${deadline}", SanitizationHelper.sanitizeForHtml(TimeHelper.formatTime12H(session.getEndTime())),
+                "${coOwnersEmails}", generateCoOwnersEmailsLine(generateCoOwnersList(course.getId())),
```

Comment:
Apologies for the oversight, fixed in https://github.com/TEAMMATES/teammates/pull/7169/commits/ff81325622a3c8b4bdabc0fa58de3a066277815a